app/tasks/maintenance/backfill_api_key_owners_task.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Maintenance::BackfillApiKeyOwnersTask < MaintenanceTasks::Task
+  def collection
+    ApiKey.where(owner_id: nil).or(ApiKey.where(owner_type: nil))
```

Comment:
Is the or necessary? I really hope not.

Reply:
I don't think so, but better safe than sorry until we can add non-null constraints